pkg/scheduler/framework/runtime/framework.go

```diff
@@ -355,6 +355,8 @@ func NewFramework(ctx context.Context, r Registry, profile *config.KubeScheduler
 		options.captureProfile(outputProfile)
 	}
 
+	// Logs Enabled and Disabled Plugins at  each extension point
+	logger.V(2).Info("Plugins enabled and disabled at each extension point", "config.Plugins", *f.ListPlugins())
```

Comment:
It only contains enabled plugins

```suggestion
	// log all enabled plugins at each extension point, taking default plugins, a given config, and multipoint into consideration.
	logger.V(2).Info("the scheduler starts to work with those plugins", "plugins", *f.ListPlugins())
```